packages/mattermost-redux/src/reducers/entities/threads/counts.ts

```diff
@@ -139,7 +139,7 @@ export function countsIncludingDirectReducer(state: ThreadsState['counts'] = {},
     case ChannelTypes.RECEIVED_CHANNEL_DELETED:
     case ChannelTypes.LEAVE_CHANNEL:
         return handleLeaveChannel(state, action, extra);
-    case ThreadTypes.RECEIVED_THREADS:
+    case ThreadTypes.RECEIVED_THREAD_COUNTS:
```

Comment:
I may be missing something (or maybe it was handled before) but shouldn't be also some changes in the websockets? When we receive a new post, or things like that?

Reply:
ThreadUpdated ws event, fires a thread read action, which then should update counts,
but it seems that although it's updating `total_unread_{threads|mentions}` counts, it's not updating `total`!
Thanks @larkox, will fix in this PR!

PS: the drawback of that would be that infinite loading could assume that there are no threads anymore to fetch, that's the only place `total` is used.

Reply:
> Thanks @larkox, will fix in this PR!

OK, it seems that ATM we have no means to derive a thread updated is a new thread or an existing one,
so we don't know whether to update the `total` count or not.

Reply:
@larkox, @hmhealey we'll need either to add a new event (ThreadCreated) or just add a prop like `new_thread: true` on `ThreadUpdated`, since this is already an issue, I want to add a new ticket for that and work on it ASAP.

Merge the other one first, since it will solve an issue in another PR as well https://github.com/mattermost/mattermost-server/pull/18492#issuecomment-1031258760